pkgs/top-level/python-packages.nix

```diff
@@ -921,13 +921,14 @@ in modules // {
 
   arrow = buildPythonPackage rec {
     name = "arrow-${version}";
-    version = "0.5.0";
+    version = "0.7.0";
 
     src = pkgs.fetchurl {
       url    = "https://pypi.python.org/packages/source/a/arrow/${name}.tar.gz";
-      sha256 = "1q3a6arjm6ysl2ff6lgdm504np7b1rbivrzspybjypq1nczcb7qy";
+      sha256 = "0yx10dz3hp825fcq9w15zbp26v622npcjscb91da05zig8036lra";
     };
 
+    # AttributeError: 'module' object has no attribute 'util_tests'
     doCheck = false;
```

Comment:
The reason you get this error is because apparently they do not support `python setup.py test` which `buildPythonPackage` uses. Therefore, to solve the issue we need to run the correct test runner manually. While it wasn't stated in their `setup.py`, their `Makefile` in the GitHub rep shows they run tests using `nosetests`. 

Therefore you need:
```
    checkPhase = ''
      nosetests
    '';

    buildInputs = with self; [ chai nose ];
```

The package `chai` isn't yet in Nixpkgs. If you could add it that would be great.
https://pypi.python.org/pypi/chai